src/charm.py

```diff
@@ -584,11 +581,22 @@ def _start_server(self):
 
             if not self._wp_is_installed():
                 self._wp_install()
-            if self._current_wp_config() is None:
-                # For security reasons, never start WordPress server if wp-config.php not exists
-                raise FileNotFoundError(
-                    "required file (wp-config.php) for starting WordPress server not exists"
+        else:
+            for _ in range(60):
+                if self._wp_is_installed():
+                    break
+                else:
+                    time.sleep(5)
+            else:
+                raise exceptions.WordPressBlockedStatusException(
+                    "leader unit failed to initialize WordPress database in given time."
                 )
+        if self._current_wp_config() is None:
+            # For security reasons, never start WordPress server if wp-config.php not exists
+            raise FileNotFoundError(
+                "required file (wp-config.php) for starting WordPress server not exists"
```

Comment:
`"required file (wp-config.php) for starting WordPress server does not exists"`?

Reply:
Fixed, thanks!